backend/data.go

```diff
@@ -161,17 +162,9 @@ type DataResponse struct {
 	Status Status
 
 	// ErrorSource is the the source of the error
-	ErrorSource ErrorSource
+	ErrorSource e.ErrorSource
 }
 
-// ErrorSource type defines the source of the error
-type ErrorSource string
-
-const (
-	ErrorSourcePlugin     ErrorSource = "plugin"
-	ErrorSourceDownstream ErrorSource = "downstream"
-)
-
```

Comment:
This is a breaking change. Should be fine given it was just introduced. Still need to be documented in SDK release.

Reply:
moved it back to backend since i removed the middleware